### PR TITLE
Unstructured mesh

### PR DESCRIPTION
**Pull request by matthewturk**
As noted in #128, this is a start on the unstructured mesh section.  I'm going to work with this to identify the best examples, and I'll build out some better viz of the elements.

### CODE REVIEW

File: content/30.abstracting_simulation_types.md [hunk 1]
```diff
@@ -159,6 +159,17 @@ For instance, this means that a "ray" object (often used to compute, for instanc
 Other than these differences, which have been intentionally made to align the results with the expected results from the underlying discretization method, the APIs for access to particle data and finite volume data are identical, and they provide broadly identical functionality, where the disparities are typically in functionality such as volume rendering.
 This allows a single analysis script, or package (such as Trident), to utilize a high-level API to address both gridded and Lagrangian data while still ensuring that the results are high-fidelity and representative of the underlying methods.
 
-### Unstructured Mesh Analysis
+### Unstructured Mesh Analysis {#sec:unstructured_mesh}
+
+`yt` has support for several different types of unstructured mesh elements.
+Typically, these are supplied as a set of coordinate points (vertices) and connectivity between those vertices.
+`yt` is able to interpret three types of elements: tetrahedral elements (4 faces, 4 vertices), wedge elements (5 faces, 6 vertices) and hexahedral elements (6 faces, 8 vertices).
```
Comment (chrishavlin) [inline content/30.abstracting_simulation_types.md:166]: Worth mentioning that 2D works -- maybe just add the following aside? 
```suggestion
`yt` is able to interpret three types of elements (and their 2D counterparts): tetrahedral elements (4 faces, 4 vertices), wedge elements (5 faces, 6 vertices) and hexahedral elements (6 faces, 8 vertices).
```

Comment (chrishavlin) [reply content/30.abstracting_simulation_types.md:166]: also, i know this is a WIP but i got excited :)

**Comment by matthewturk** [comment]
Still trying to decide if I should include more info from `mesh_types.yaml` or leave it there.  Going to add example plots next.

**Comment by matthewturk** [comment]
I thought at first that a pretty-printed version of the interpolation functions would be nice, but I'm not convinced it adds much value.

image

I do think it might be reasonable to add in small 3D figures showing them, though, so I will look at that.  @atmyers what do you think about the equations -- is it worth adding them in, or even adding just the shape functions from the yaml file, or is that unnecessary since they're available elsewhere?

**Comment by matthewturk** [comment]
I'm going to call this good.  I developed a finite element viewer in threejs, and a plugin for it, but I do not believe it materially adds to the discussion in a way that is not better served by other presentations.

**Pull request merged**